lib/features.js

```diff
@@ -52,6 +53,23 @@ module.exports = config => {
     },
 
     /**
+     * Return whether an account meets the account created age requirement to
+     * bypass sign-in confirmation.
+     *
+     * @returns {boolean}
+     */
+    isSigninBypassAccountWithAgeEnabled(account) {
+      const bypassAccountWithAge = signinConfirmation.bypassAccountWithAge
+      var accountCreatedSince = Date.now() - account.createdAt
+      if (bypassAccountWithAge && bypassAccountWithAge.enabled
+        && accountCreatedSince < bypassAccountWithAge.accountCreatedSinceMS) {
+        return true
+      }
+
+      return false
```

Comment:
This can just return the boolean result of the `if` on line 64.

```js
return !!bypassAccountWithAge && bypassAccountWithAge.enabled && accountCreatedSince < bypassAccountWithAge.accountCreatedSinceMS
```